public/modules/custom/infofinland_user_cancel/infofinland_user_cancel.module

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Not sure if it makes sense to make a module just for this. This could be moved to `infofinland_admin_tools` module instead.

Reply:
This should make it easier to adjust the weight later if some other modules causes problems by invoking it's "user_cancel" hook first. Also makes it easier to disable this module quickly if it turns out to cause problems down the line. Both might cause issues with the other functionality in `infofinland_admin_tools`.